dali/operators/math/normalize/normalize_utils.h

```diff
@@ -136,6 +136,9 @@ static void ScaleRSqrtKeepZero(const TensorView<StorageCPU, float> &inout, float
 static void SumSquare2InvStdDev(const TensorView<StorageCPU, float> &inout,
                                 const TensorShape<> &data_shape,
                                 double scale) {
+  if (inout.num_elements() == 0 || data_shape.num_elements() == 0) {
```

Comment:
```suggestion
  if (inout.num_elements() == 0)
    return;
  assert(data_shape.num_elements() >= inout.num_elements());
```

Reply:
assert covers only a debug build case, but we have it secured for the release build.

Reply:
It's not "secured", it's still a misuse and an error to call it with zero-volume data_shape. We don't want to hide such errors, should they arise.
`data_shape` is the shape of the original (non-reduced) data, and therefore there's a strict relation between `data_shape` and `inout.shape()`:
For all valid values of i  `inout.shape[i] ==data_shape[i] || inout.shape[i] == 1`.
If this condition is not met, it's a _coding error_ at call site. This should never happen.
Having that said, it seems there actually _is_ a bug in the operator - we don't skip empty tensors (normalize.cc:248) and we should.

Reply:
Done